electron-builder.env

```diff
@@ -1 +1,2 @@
 BUNDLING=true
+CI=true
```

Comment:
This env variable is something that most CI tools set automatically, and not needed locally imo.